### DISCUSSION TITLE
Remove locale dropdown

### DISCUSSION THREAD

**Discussion by chmcquee** [keycloakify/keycloakify#236] [Q&A] [answered]
Hiya,
so I'm working on a project where I want to remove the language dropdown by default. 
I'm using the component level customisation for the login screen. 
Is there any way to do this, without having to customise the entire Template file?

Thanks a lot,
Chris

**Comment by garronej** [comment, marked as answer]
Hi @chmcquee,  
There is a handle on `kc-locale-wrapper`, you can add  `display: none;` or something like this  
<img width="670" alt="image" src="https://user-images.githubusercontent.com/6702424/216083595-1a830223-e063-4c0c-b90c-effcbc9033e6.png">

https://github.com/garronej/keycloakify-starter/blob/a01ca67cb5bae548a5dea80e330c40afe289e2f8/src/KcApp/KcApp.tsx#L63-L67

**Comment by lordvlad** [comment]
If you change the number of supported locales to be a single one, it will
be hidden by default (see the first line in the screenshot,
locale.supported.length > 1), no need to mess with the theme for that.
